backend/ibutsu_server/__init__.py

```diff
@@ -64,7 +71,19 @@ def get_app(**extra_config):
                     port=config.get("POSTGRESQL_PORT"),
                     user=config.get("POSTGRESQL_USER"),
                     password=config.get("POSTGRESQL_PASSWORD"),
-                )
+                ),
+                "CELERY_BROKER_URL": _make_broker_url(
+                    config.get("CELERY_BROKER_URL"),
+                    config.get("REDIS_HOSTNAME"),
+                    config.get("REDIS_PASSWORD"),
+                    config.get("REDIS_PORT"),
+                ),
+                "CELERY_RESULT_BACKEND": _make_broker_url(
+                    config.get("CELERY_RESULT_BACKEND"),
+                    config.get("REDIS_HOSTNAME"),
+                    config.get("REDIS_PASSWORD"),
+                    config.get("REDIS_PORT"),
+                ),
```

Comment:
Can we move these into their own `if` block? 

```python
if config.get("CELERY_BROKER_URL") or config.get("REDIS_HOSTNAME"):
```

```python
if config.get("CELERY_RESULT_BACKEND") or config.get("REDIS_HOSTNAME"):
```

Reply:
Ah, didn't catch that. Yep.